gl_generator/lib.rs

```diff
@@ -1,68 +1,32 @@
-// Copyright 2015 Brendan Zabarauskas and the gl-rs developers
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
+// Based on the `gl_generator` crate, by Brendan Zabarauskas and the gl-rs
+// developers (2015). Used under Apache-2.0 license. All new content is also
+// available under the Apache-2.0 license.
```

Comment:
Do we need to change this? I'm guessing this was from the previous fork? I'm cool if we remove this now like in the other file - I know now that "the gl-rs developers" isn't really a valid 'legal entity'.

Reply:
oh! right, that's from when i made this a stand alone crate. i will revert it in the next series of revisions.